fix: tasks on timeline not adjusting to zoom

JosephBrooksbank opened the pull request:
#243 

There are issues in svelte with cascading dependencies in reactive declarations, from what I understand.
Since `background color` relies on `$store.timelineColored`, having `textColor` rely on both seemed to cause issues. Removing the direct dependency of the store works, anyway.

Fix gif: 
![zoom](https://github.com/ivan-lednev/obsidian-day-planner/assets/6687530/b15e6a1c-d791-452e-9247-375202793764)

JosephBrooksbank committed:
fix: tasks on timeline not adjusting to zoom

There are issues in svelte with cascading dependencies
in reactive declarations, from what I understand.
Since `background color` relies on `$store.timelineColored`,
having `textColor` rely on both seemed to cause issues.
Removing the direct dependency of the store works,
anyway.

ivan-lednev commented:
Hello, @JosephBrooksbank. This is not related to cascading deps, since you only read settings, and don't update them in one of your reactive blocks.

It's because you're mixing different inline style approaches and using auto-subscriptions:
```
style="foo: {$bar};"
style:baz={$bazz}
```

I managed to narrow it down: https://svelte.dev/repl/3806195fc2924034a9175a8d38dedc87?version=4.2.0

I guess I'll have to create a bug in Svelte repo. I've already lost count of how often I regretted not picking React or Solid over Svelte 😆 

Try setting the variables directly from Task on RenderedMarkdown using this syntax: https://svelte.dev/docs/component-directives#style-props

JosephBrooksbank committed:
fix: consistent style tag usage

https://github.com/ivan-lednev/obsidian-day-planner/pull/244#issuecomment-1708985957

JosephBrooksbank commented:
Interesting- I didn't really think I had a cascading issue either, but it was the first thing I considered and removing the second reference to the store worked, so I didn't question it 😅 any thoughts on why the original solution also fixes it? 

I've updated it to use consistent inline styles- I'm happier with this overall, I wanted to use that style directive from the start, but you can't put it on raw divs, and we were using color vars in `task`. Now that they're all in `RenderedMarkdown`, it works great
